best-practices/README.md

```diff
@@ -215,24 +215,34 @@ HTML
 * Don't use a reset button for forms.
 * Prefer cancel links to cancel buttons.
 * Use `<button>` tags over `<a>` tags for actions.
+* Use semantic tags where applicable. 
 
 CSS
 ---
 
 * Use Sass.
-* Use [Autoprefixer][autoprefixer] to generate vendor prefixes based on the
-  project-specific browser support that is needed.
-
-[autoprefixer]: https://github.com/postcss/autoprefixer
 
 Sass
 ----
 
-* Prefer `overflow: auto` to `overflow: scroll`, because `scroll` will always
-  display scrollbars outside of OS X, even when content fits in the container.
 * Use `image-url` and `font-url`, not `url`, so the asset pipeline will re-write
   the correct paths to assets.
-* Prefer mixins to `@extend`.
+* Prefer mixins over `@extend`.
+* Prefer reusable classes over mixins.
+* Avoid using ID selectors.
+* Avoid element selectors.
+* Avoid decendant selectors, or any highly specific selectors when possible.
+* Avoid nesting
```

Comment:
Tiny nitpick: most lines end with a `.`, but this one doesn't (also 242, and 243)